tests/test-update-from-main.c

```diff
@@ -135,6 +136,12 @@ test_update_from_main (EosUpdaterFixture *fixture,
                                  &branches_option, &error);
   g_assert_no_error (error);
   g_assert_cmpstr (branches_option, ==, expected_branches);
+
+  ostree_repo_get_remote_option (repo, default_remote_name,
+                                 "collection-id", NULL,
+                                 &collection_id_option, &error);
+  g_assert_no_error (error);
+  g_assert_cmpstr (collection_id_option, ==, default_collection_ref->collection_id);
```

Comment:
Does this actually test that the collection ID has been set, and wasn’t set before? From a quick reading of the code, since `default_collection_ref` is non-`NULL`, it looks like `eos_test_client_new()` will set the collection ID in the repo `config` when it’s first created.

Reply:
Pushed a fixup so that we do check the transition from NULL to non-NULL